power_grid_model_c/power_grid_model/include/power_grid_model/auxiliary/serialization/deserializer.hpp

```diff
@@ -703,7 +703,7 @@ class Deserializer {
     void parse_component(Idx component_idx) {
         if (dataset_handler_.is_row_based(component_idx)) {
             parse_component(row_based, component_idx);
-        } else {
+        } else if (dataset_handler_.is_columnar(component_idx, true)) {
```

Comment:
this skips components without any buffers provided by the user

Reply:
that's good improvement